Neurolab.cpp

```diff
@@ -0,0 +1,100 @@
+#include "Neurolab.h"
+
+AD7173Class ads;
+  
+// VARIABLES
+int  outputType;
+int auxData[3];
+boolean useAux;
+
+// BOARD WIDE FUNCTIONS
+void Neurolab::initialize(void){
+  /* initiate serial communication */
+  Serial.begin(230400);
+        
+  /* initialize SPI connection to the ADC */
+  initialize_adc();
+
+  /* sync the ADC */
+  sync_adc();
+}
+
+/*
+==================================================
+TO-DO list
+==================================================
+*/     
+void Neurolab::updateChannelSettings(void){
+}
```

Comment:
Hi, Please add new lines after each method. Thanks

Reply:
Hi, thanks for the review. I'll add a new line with  `/* ... */` since it's the coding style used in AD7173.cpp